modules/cli/cmd/version.go

```diff
@@ -22,7 +23,12 @@ func printVersion() {
 	if version == "" {
 		version = "development"
 	}
-	fmt.Println(version)
+	fmt.Println("astria-go:", version)
```

Comment:
I think this should actually be its own command under devrunner, because now we're muddling the overall cli with the devrunner functionality. 

`astria-go dev versions` maybe